frontend/src/pages/home/components/live_meeting/LiveMeeting.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react'
+import { useNavigate } from 'react-router-dom';
+import { getRelativeTime } from '../meetingUtils';
+import MeetingDTO from '../../../../shared/Types/dtos/MeetingDTO';
+import './livemeeting.css'
+
+export default function LiveMeeting(props: { meeting: MeetingDTO }) {
+
+    const [timeLeft, setTimeLeft] = useState<string>(getRelativeTime(props.meeting.time + 3_600_000))
+
```

Comment:
You don't need to specify that the state is a `string` here, Typescript should be smart enough to infer that from the return type of the method :)